src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCode.cs

```diff
@@ -210,7 +210,8 @@ public static int OpcodeSize(int opcode)
                     return 3;
 
                 default:
-                    throw new ArgumentException(SR.Format(SR.UnexpectedOpcode, opcode.ToString()));
+                    Debug.Fail($"Unexpected opcode: {opcode}");
```

Comment:
Is there a specific reason why we decided to not throw in this case any longer?

Reply:
It's unreachable. We'd only get here if there was a missing case.